### PR TITLE
Feature/character card component

### PR DESCRIPTION
**Pull request by AchimBartscht**
Add "Character Card Component"

### CODE REVIEW

**Review by willywdev — APPROVED** [review]
Looks really clean!
Append in index.js and return the newCard in the function.
Will work better in future challenges

File: components/card/card.js [hunk 1]
```diff
@@ -1 +1,27 @@
...
+      <dd class="card__info-description">${character.status}</dd>
+      <dt class="card__info-title">Type</dt>
+      <dd class="card__info-description">${character.type}</dd>
+      <dt class="card__info-title">Occurrences</dt>
+      <dd class="card__info-description">${character.episode.length}</dd>
+    </dl>
+  </div>`;
+  cardContainer.append(newCard);
+}
+createCharacterCard();
```
Comment (ahohnsen) [inline components/card/card.js:27]: Please follow the instructions in the readme file :) The `createCharacterCard` function should return your `newCard`. In the `index.js` you will need to import the `createCharacterCard` function and call it there.
```suggestion
  return(newCard);
}

```

File: components/card/card.js [hunk 2]
```diff
@@ -1 +1,27 @@
-export function createCharacterCard() {}
+const cardContainer = document.querySelector('[data-js="card-container"]');
```
Comment (ahohnsen) [inline components/card/card.js:1]: As you should not append the card in this file, you won't need the `cardContainer` here. See my other comment below. :) 
```suggestion

```

File: components/card/card.js [hunk 3]
```diff
@@ -1 +1,27 @@
...
+export function createCharacterCard(character) {
```
Comment (ahohnsen) [inline components/card/card.js:2]: As you only export this function from this file, we usually use `export default` for it.
```suggestion
export default function createCharacterCard(character) {
```

File: index.js [hunk 4]
```diff
@@ -1,3 +1,4 @@
+import { createCharacterCard } from "./components/card/card.js";
```
Comment (ahohnsen) [inline index.js:1]: This is going to be a default export (see my comment above), so you will need to adjust this import too:
```suggestion
import createCharacterCard from "./components/card/card.js";
```

**Review by ahohnsen — CHANGES_REQUESTED** [review]
Heyho!

I left some comments for you 🙂 Please have a look.
Additionally, please do not forget to delete the `li`-element inside your `index.html` file as you are now creating all `li`-elements dynamically with JavaScript.

**Pull request merged**